includes/OutputPage.php

```diff
@@ -2745,6 +2745,8 @@ public function getResourceLoader() {
 	public function makeResourceLoaderLink( $modules, $only, $useESI = false, array $extraQuery = array(), $loadCall = false ) {
 		global $wgResourceLoaderUseESI;
 
+		$modules = (array) $modules; // Wikia change - PHP 7.3
```

Comment:
Can `$modules` be declared as an `array` instead of this cast?

Reply:
I've simply "backported" the approach from MediaWiki's upstream - https://github.com/wikimedia/mediawiki/blob/c4e284f1130cdc2a4811120594bdb8013d17eaaa/includes/OutputPage.php#L3303

But sure, let's try adding a type hint in method declaration.

Reply:
I just read the comment for this method, and `$modules` is an `array` or `string`, so this is the only choice.